tests/FakeItEasy.Tests/Core/FakeWrapperConfiguratorTests.cs

```diff
@@ -1,9 +1,12 @@
 namespace FakeItEasy.Tests.Core
 {
     using System.Linq;
+    using System.Reflection;
     using FakeItEasy.Core;
     using FakeItEasy.Creation;
+#if FEATURE_BINARY_SERIALIZATION

```

Comment:
Did you copy-n-pasted a wrong one?  I submitted https://github.com/jeremymeng/FakeItEasy/pull/4